napari/_qt/_qapp_model/qactions/_help.py

```diff
@@ -0,0 +1,41 @@
+"""This module defines help menu actions.
+"""
+from typing import List
+
+from app_model.types import Action, KeyBindingRule, KeyCode, KeyMod
+
+from ...._app_model.constants import CommandId, MenuId
+from ....utils.translations import trans
+from ...dialogs.qt_about import QtAbout
+from ...qt_main_window import Window
+
+try:
+    from napari_error_reporter import ask_opt_in
+except ModuleNotFoundError:
+    ask_opt_in = None
+
+
+def _show_about(window: Window):
+    QtAbout.showAbout(window._qt_window)
+
+
+HELP_ACTIONS: List[Action] = [
+    Action(
+        id=CommandId.NAPARI_INFO,
+        title=CommandId.NAPARI_INFO.title,
+        callback=_show_about,
+        menus=[{"id": MenuId.MENUBAR_HELP}],
+        status_tip=trans._('About napari'),
+        keybindings=[KeyBindingRule(primary=KeyMod.CtrlCmd | KeyCode.Slash)],
```

Comment:
interestingly this keybinding only brought up the help menu on macOS, and not the about dialog. I assume this is another Qt heuristics because it also added a search bar...